requirements.in

```diff
@@ -24,5 +24,6 @@ Jinja2
 jsonschema
 premailer==2.9.2
 schematics==1.0.4
+scrapinghub==1.9.0
```

Comment:
should it be mentioned in requirements.in? it's a `scrapinghub-entrypoint-scrapy` requirement after all...